apps/ai-dial-admin/src/components/ActivityAudit/View/DiffReport/DiffLegend.tsx

```diff
@@ -15,27 +15,30 @@ interface Props {
 const DiffLegend: FC<Props> = ({ description, added, removed, changed }) => {
   const t = useI18n();
 
-  const containerClass = classNames('flex flex-row small', description ? 'gap-8 text-secondary' : 'gap-2 text-primary');
-  const itemClass = classNames('flex items-center gap-1');
-  const descriptionClass = classNames(description ? '' : 'hidden');
-  const rectangleClass = classNames(
+  const containerClassName = classNames(
+    'flex flex-row small',
+    description ? 'gap-8 text-secondary' : 'gap-2 text-primary',
+  );
+  const itemClassName = classNames('flex items-center gap-1');
+  const descriptionClassName = classNames(description ? '' : 'hidden');
+  const rectangleClassName = classNames(
     'inline-block rounded-sm border',
     description ? 'w-[14px] h-[14px]' : 'w-[24px] h-[24px] flex items-center justify-center',
   );
 
   return (
-    <div className={containerClass}>
-      <div className={`${itemClass} ${(description ? false : !added) && 'hidden'}`}>
-        <span className={`${rectangleClass}  bg-success border-accent-secondary`}>{added}</span>
-        <span className={descriptionClass}>{t(ButtonsI18nKey.Create)}</span>
+    <div className={containerClassName}>
+      <div className={`${itemClassName} ${(description ? false : !added) && 'hidden'}`}>
+        <span className={`${rectangleClassName}  bg-success border-accent-secondary`}>{added}</span>
```

Comment:
use `classNames` instead of template literals/string interpolation here and in all other places
```suggestion
        <span className={classNames(rectangleClassName, bg-success border-accent-secondary`)}>{added}</span>
```

I would suggest to find it by "className={`"